### ISSUE TITLE
Linux fonts in episodes header too small on Linux (?)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#280]
```
Look at screenshot, the fonts are too small in Linux - don't know how this is 
at Mac?
```

Original issue reported on code.google.com by `christian.a.reiter@gmail.com` on 22 Jan 2013 at 8:53

Attachments:
- [Auswahl_005.png](https://storage.googleapis.com/google-code-attachments/freemedforms/issue-280/comment-0/Auswahl_005.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `eric.mae...@gmail.com` on 22 Jan 2013 at 8:59
- Changed state: **Started**
- Added labels: **Milestone-Release0.8.0**, **Priority-Critical**
- Removed labels: **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision e1ccb7fc4dbf.
```

Original comment by `eric.mae...@gmail.com` on 22 Jan 2013 at 9:51
- Changed state: **Fixed**
